app/uk/gov/hmrc/cdsreimbursementclaim/controllers/DeclarationInfoController.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsreimbursementclaim.controllers
+
+import javax.inject.{Inject, Singleton}
+import play.api.libs.json.JsValue
+import play.api.mvc.{Action, ControllerComponents}
+import uk.gov.hmrc.cdsreimbursementclaim.services.DeclarationInfoService
+import uk.gov.hmrc.cdsreimbursementclaim.utils.Logging
+import uk.gov.hmrc.cdsreimbursementclaim.utils.Logging.LoggerOps
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+
+import scala.concurrent.ExecutionContext
+
+@Singleton()
+class DeclarationInfoController @Inject()(declarationInfoService: DeclarationInfoService, cc: ControllerComponents)(
+  implicit ec: ExecutionContext) extends BackendController(cc) with Logging {
+
+  def getDeclarationInfo(): Action[JsValue] = Action.async(parse.json) { implicit request =>
+    declarationInfoService
+      .getDeclarationInfo(request.body)
+      .fold(
+        e => {
+          logger.warn(s"could not get declaration information", e)
+          InternalServerError
+        },
+        response => Ok(response)
```

Comment:
When you change the service layer method to construct the actual data structure from the response from ACC14 then you will need to serialise the resultant data structure here.